### PR TITLE
Elbencho: add elbencho package

### PR DESCRIPTION
**Pull request by ethanjjjjjjj**
Add package for elbencho storage benchmark

https://github.com/breuner/elbencho

### CODE REVIEW

File: var/spack/repos/builtin/packages/elbencho/package.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    def edit(self, spec, prefix):
+        os.mkdir(prefix.bin)
+        os.environ["INST_PATH"] = prefix.bin
+        if "+s3" in spec:
+            os.environ["S3_SUPPORT"] = "1"
+        if "+cuda" in spec:
+            os.environ["CUDA_SUPPORT"] = "1"
+        if "+cufile" in spec:
+            os.environ["CUDA_SUPPORT"] = "1"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/elbencho/package.py]: Is `CUDA_SUPPORT` the right variable when `cufile`is enabled?

**Review by tldahlgren — COMMENTED** [review]
Confirmed the homepage and 8 version sha256.  Do have a question about the environment variable associated with one of the variants.

File: var/spack/repos/builtin/packages/elbencho/package.py (hunk 1 above)
Comment (ethanjjjjjjj) [reply var/spack/repos/builtin/packages/elbencho/package.py]: Good catch, fixed that.

**Review by tldahlgren — APPROVED** [review]
I'm able to successfully build the package with defaults.

**Pull request merged**